packages/@uppy/provider-views/src/utils/PartialTreeUtils/afterFill.ts

```diff
@@ -6,6 +6,7 @@ import type {
   PartialTreeId,
 } from '@uppy/core/lib/Uppy.js'
 import type { CompanionFile } from '@uppy/utils/lib/CompanionFile'
+// eslint-disable-next-line import/no-unresolved
```

Comment:
So it seems it's not working, no? What changed since https://github.com/transloadit/uppy/pull/4530#issuecomment-1743119329?

Reply:
it seems to only affect eslint (as there are no build errors in this PR). I think it's better to not have this patch because if we have this patch we might not discover build issues that might arise from this package in the future. and why have a patch if everything works without it?

Reply:
Maybe we can patch the eslint plugin (and then send the patch upstream)?

Reply:
feel free 🙌

Reply:
So the problem is that it uses `resolve`, which does not support the modern resolution at all.
```suggestion
// p-queue does not have a `"main"` field in its `package.json`, and that makes `import/no-unresolved` freak out.
// We can safely ignore it because bundlers will happily use the `"exports"` field instead.
// eslint-disable-next-line import/no-unresolved
```

Reply:
ok nice find